specific/rje_purchase/models/purchase.py

```diff
@@ -12,3 +12,6 @@ class PurchaseOrder(models.Model):
         states={'draft': [('readonly', False)],
                 'sent': [('readonly', False)]},
     )
+    authorized_id = fields.Many2one('hr.employee',
```

Comment:
Can you change field name to -> authorized_employee_id.
Our rule is to define name that is other can instantly understand it.

for authorized_id, I won't know which model it is from. But authorized_employee_id, I will know it is an employee.